static/js/message_edit.js

```diff
@@ -273,9 +274,12 @@ function edit_message (row, raw_content) {
         var original_topic = message.subject;
         topic_input.keyup( function () {
             var new_topic = topic_input.val();
-            row.find('.message_edit_topic_propagate').toggle(new_topic !== original_topic);
+            row.find('.message_edit_topic_propagate').toggle(new_topic !== "");
```

Comment:
ah, so we still want to keep the old behavior as well here, something like 
row.find(...).toggle((new_topic !== original_topic) && (new_topic !== ""))